### PR TITLE
fix: Markdownlint cleanup for ES

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 1]
```diff
@@ -87,13 +87,13 @@ Ya hemos visto el elemento {{htmlelement("title")}} en acción — se puede usar
 
 2. Ahora abre el archivo en tu navegador. Deberías ver algo como esto:
 
-   ![Una sencilla página web con el título configurado a <title> element, y el <h1> configurado a <h1> element.](https://mdn.mozillademos.org/files/12323/title-example.png)Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
+   ![Una sencilla página web con el título configurado a \<title> element, y el \<h1> configurado a \<h1> element.](https://mdn.mozillademos.org/files/12323/title-example.png)Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
```
Comment (JuanVqz) [inline files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:90]: The image is not rendered. 

```suggestion
   ![Una sencilla página web con el título configurado a \<title> element, y el \<h1> configurado a \<h1> element.](title-example.png)
   Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
```

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 2]
```diff
@@ -87,13 +87,13 @@ Ya hemos visto el elemento {{htmlelement("title")}} en acción — se puede usar
...
 
 3. También podrías probar a abrir el código en tu editor de código, editar el contenido de estos elementos y luego actualizar la página en tu navegador. Juega un poco con ello.
 
 El contenido del elemento `<title>` también se usa de otras formas. Por ejemplo, si intentas marcar la página como favorita (**Marcadores → Marcar esta página**, o el icono en forma de estrella de Firefox), verás que el nombre que se sugiere para el marcado es el contenido del elemento `<title>`.
 
-![Se está marcando una página web en Firefox; el nombre del marcador se ha completado automáticamente con el contenido del elemento <title>](https://mdn.mozillademos.org/files/12337/bookmark-example.png)
+![Se está marcando una página web en Firefox; el nombre del marcador se ha completado automáticamente con el contenido del elemento \<title>](https://mdn.mozillademos.org/files/12337/bookmark-example.png)
```
Comment (JuanVqz) [inline files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:96]:
```suggestion
![Se está marcando una página web en Firefox; el nombre del marcador se ha completado automáticamente con el contenido del elemento \<title>](bookmark-example.png)
```

**Review by JuanVqz — APPROVED** [review]
There are more images that are not rendered, If you feel confident, can you please fix that in another PR?

Also, please send one document per PR, :pray: thanks!

image

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md (hunk 1 above)
Comment (JuanVqz) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:90]: Commit this suggestion is required.

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md (hunk 2 above)
Comment (JuanVqz) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:96]: Commit this suggestion is required.

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md (hunk 1 above)
Comment (nschonni) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:90]: I believe those files were brought in for the en-US content, but that file isn't local in this repo https://github.com/mdn/translated-content/tree/main/files/es/learn/html/introduction_to_html/the_head_metadata_in_html

**Comment by nschonni** [comment]
> Also, please send one document per PR, 🙏 thanks!

The Markdownlint cleanup is done in batches, otherwise it would create hundreds of PRs per local. There is an open PR that would add it to the CI, so these issues will be addressed before they're introduced though. It's just "messy" right now during the conversion.

File: files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md (hunk 1 above)
Comment (JuanVqz) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:90]: yes, it is inherited from the `content` repository, yari takes care of that.

Comment (JuanVqz) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md]: but let's fix it on other pr, thanks for contributing!

Comment (nschonni) [reply files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md]: OK, i've added those changes, but it's probably better to do a big cleanup once the Markdown conversion is done

**Pull request merged**